content/plugin/framework/acctests.mdx

```diff
@@ -7,12 +7,12 @@ description: >-
 
 # Acceptance Tests
 
-The framework for Terraform provider development currently relies on the
-[acceptance test framework](/plugin/sdkv2/testing/acceptance-tests)
-shipped with SDKv2. You will write and run tests with the same steps, and they
-are otherwise indistinguishable from testing against SDKv2.  The major
-difference is in how the provider is specified in [the `TestCase`
-struct](/plugin/sdkv2/testing/acceptance-tests/testcase).
+Testing can be implemented using the [acceptance test framework](/plugin/sdkv2/testing/acceptance-tests) shipped with SDKv2.
```

Comment:
```suggestion
You can implement testing with the [acceptance test framework](/plugin/sdkv2/testing/acceptance-tests) shipped with SDKv2.
```

Reply:
I swear, one of these days, I will properly write this type of documentation with active voice! My apologies you always need to nudge me in that direction. 😅